ArduCopter/ekf_check.cpp

```diff
@@ -296,7 +297,7 @@ void Copter::check_vibration()
             vibration_check.high_vibes = true;
             pos_control->set_vibe_comp(true);
             LOGGER_WRITE_ERROR(LogErrorSubsystem::FAILSAFE_VIBE, LogErrorCode::FAILSAFE_OCCURRED);
-            gcs().send_text(MAV_SEVERITY_CRITICAL, "Vibration compensation ON");
```

Comment:
Hi, yes, this is fine but could you change this to use GCS_SEND_TEXT?  Also we don't use Pascal case normally so if you could rename it to "vibration_compensation_str" that'd be nice.  After that we can merge.

Reply:
Also I'd actually prefer if we did it like this:

static const char *vibration_compensation_str = "Vibration compensation %s";
GCS_SEND_TEXT("MAV_SEVERITY_CRITICAL, "%s ON", vibration_compensation_str);